components/src/Select/Select.js

```diff
@@ -0,0 +1,175 @@
+  lineHeight: theme.lineHeights.base,
+  border: "1px solid",
+  borderColor: getBorderColor(error, isOpen, disabled, false),
+  borderTopLeftRadius: theme.radii[1],
+  borderTopRightRadius: theme.radii[1],
+  borderBottomLeftRadius: isOpen ? 0 : theme.radii[1],
+  borderBottomRightRadius: isOpen ? 0 : theme.radii[1],
+  boxShadow: isOpen ? theme.boxShadows[0] : "none",
+  outline: "none",
+  background: disabled ? theme.colors.whiteGrey : theme.colors.white,
+  "&:hover, &:focus": {
+    cursor: "default",
+    borderColor: getBorderColor(error, isOpen, disabled, true),
+  },
+}));
+
+const IndicatorButton = styled.div([], () => ({
+  position: "absolute",
+  top: theme.space[2],
+  right: theme.space[1],
+  bottom: "0",
+  pointerEvents: "none",
+}));
+
+const ToggleButton = isOpen => (
+  <IndicatorButton>
+    {
+      isOpen
+        ? <Icon icon="upArrow" />
+        : <Icon icon="downArrow" />
+    }
+  </IndicatorButton>
+);
+
+ToggleButton.propTypes = {
+  isOpen: PropTypes.bool.isRequired,
+};
+
+const Menu = styled.div([], ({ error, disabled }) => ({
+  position: "absolute",
+  width: "100%",
+  borderWidth: "1px",
+  borderColor: getBorderColor(error, true, disabled, false),
+  borderBottomStyle: "solid",
+  borderLeftStyle: "solid",
+  borderRightStyle: "solid",
+  borderRadius: `0 ${theme.radii[1]}`,
+  marginTop: 0,
+  boxShadow: theme.boxShadows[0],
+  background: disabled ? theme.colors.whiteGrey : theme.colors.white,
+  zIndex: 100,
+}));
+
+const MenuItem = styled.div([], ({ isSelected, isActive }) => ({
+  color: theme.colors.black,
+  padding: subPx(theme.space[2]),
+  fontWeight: isSelected ? "bold" : "normal",
+  background: isActive ? theme.colors.lightBlue : null,
+  "&:hover": {
+    background: theme.colors.lightBlue,
+  },
+  "&:last-child": {
+    borderRadius: `0 ${theme.radii[1]}`,
+  },
+}));
+
+const Select = ({
+  error, onChange, disabled, options, optionToString, value, required, placeholder, initialIsOpen
+}) => (
+  <Downshift
+    itemToString={ optionToString }
+    selectedItem={ value }
+    onChange={ onChange }
+    defaultHighlightedIndex={ 0 }
+    initialIsOpen={ initialIsOpen }
+  >
+    {
+      ({
+        getMenuProps,
+        getItemProps,
+        getInputProps,
+        getToggleButtonProps,
+        isOpen,
+        selectedItem,
+        highlightedIndex,
+      }) => (
+        <div style={ { position: "relative" } }>
+          <SelectBox { ...getToggleButtonProps({ disabled, error, isOpen }) }>
+            <Input
+              { ...getInputProps({ disabled, error, isOpen }) } aria-required={ required }
+              value={ selectedItem && selectedItem.label } placeholder={ placeholder }
+              readOnly
+            />
+            <ToggleButton isOpen={ isOpen } />
+          </SelectBox>
+          {
+            isOpen
+              && (
+                <Menu { ...getMenuProps({ error }) }>
+                  {
+                    isOpen
```

Comment:
You can probably drop this inner conditional render, the outer one is already based on isOpen anyway